test/integration/040_compare_test/test_compare.py

```diff
@@ -2,13 +2,13 @@
 import os
 
 
-class TestConfigs(DBTIntegrationTest):
+class TestCompare(DBTIntegrationTest):
     @property
     def schema(self):
         return "config_040"
 
     def unique_schema(self):
-        return super(TestConfigs, self).unique_schema().upper()
+        return super(TestCompare, self).unique_schema().upper()
```

Comment:
blegh, just remove the `upper` here - my mistake!

Reply:
Ahhhhhhh